clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/ThreadPoolDefinition.java

```diff
@@ -37,4 +40,8 @@
     Attribute getQueueLength();
 
     Attribute getKeepAliveTime();
+
+    PathElement getPathElement();
+
+    Collection<Attribute> getAttributes();
```

Comment:
Why is this necessary?

Reply:
Fixed, removed.

Reply:
Originally needed to support both thread pool definitions in generic writer method.